index.html

```diff
@@ -177,7 +181,7 @@ <h3>Itachi Uchiha</h3>
 
     <footer>
       <div class="footer">
-        <img src="./images/cc.logo.white.svg" alt="">
+        <img src="./images/cc.logo.white.svg" alt="" />
```

Comment:
Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image

Reply:
Done. Thank you for your reviews